sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -501,7 +501,7 @@ INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_re
 
```

Comment:
Finalized thoughts (to avoid replicating entries):
place the following here before the below IfNotRow4D:
``````
UPDATE `supported_external_dataloads` SET `load_filename`='2018-ICD-10-Code-Descriptions.zip' WHERE `load_filename`='2018-ICD-10-Code-Dedcriptions.zip' AND `load_release_date`='2017-10-01';